lib/Future.h

```diff
@@ -138,6 +138,8 @@ class Promise {
 
     bool setFailed(Result result) const { return state_->complete(result, {}); }
 
+    bool setSuccess() const { return state_->complete({}, {}); }
```

Comment:
I think we can replace this method with a `setValue({})` call or passing a static variable to `setValue`.